index.js

```diff
@@ -27,9 +27,10 @@ module.exports = async (opts) => {
 
   const outputPath = path.parse(output)
 
-  if (typeof ffmpegPath !== 'undefined' && ffmpegPath !== '') {
+  if (ffmpegPath) {
```

Comment:
@smagne87 note that this updated version does the same thing as your original version. It's just a bit simpler.